code/mobility-services/mobility-services.css

```diff
@@ -228,6 +237,53 @@ a.big-button-container {
 a.small-button {
   text-decoration: none;
 }
+/****************************************/
+/*** Dropdown Menu Buttons Navigation ***/
+/****************************************/
+
+/* hide custom menu if on mobile */
+@media (max-width: 799px) {
+  #tia-menu-list {
+    display: none;
+  }
+}
+
+/* hide knack menu buttons if on desktop */
+@media (min-width: 799px) {
+  #view_163 {
+    display: none;
+  }
+}
+
+/*list, list item, and button stylings*/
+#oa-menu-list {
+  border-bottom: 0 !important
+}
+
+#oa-menu-list li {
+  margin-right: .5em;
+  box-shadow: 0px 2px 4px 0px gray;
+}
+
+#oa-menu-list .kn-button a {
+  color: #163f6e;
+  font-size: 1.1em;
+  border-bottom: 0;
+  padding-top:3px;
+}
+
+.oa-dropdown-menu-list .kn-button {
+  border-radius: 0;
+}
+
+.oa-dropdown-menu-list li {
+  border: 0;
+  margin-right: 0 !important;
+} 
+
+.oa-dropdown-menu-list li:hover {
+  background-color: rgb(235,235,235);
+}
```

Comment:
seems like the rest of the mobile stylings are missing

```
/*Mobile list, list item, and button stylings*/
.mobile-details-dropdown-menu ul {
  margin-left: .25em;
}

#mobile-menu-list {
  list-style-type: none;
  margin: 0;
} 

#mobile-menu-list ul {
  list-style-type: none;
}

.mobile-dropdown-menu span {
  margin-top: .25em;
  color: #163f6e;
}

.mobile-dropdown-button {
  display: flex;
}

.mobile-dropdown-button i {
  display: none;
}

.mobile-dropdown-button.show-icon i {
  display: inherit;
  width: 16px;
}

.mobile-dropdown-button:not(.show-icon)::before {
  content: "\203A";
  font-size: 18px;
  font-weight: 800;
  width: 16px;
  line-height: 16px;
}

.mobile-dropdown-menu-list {
  margin-top: 1em;
  display: none;
}

.mobile-dropdown-menu-list.active {
  display: inherit;
}

.desktop-button {
  background-color: rgba(0,0,0,.08);
  color: #163f6e;
  font-size: 1.1em;
  padding: .4em .8em;
  height: 32px;
  border-radius: .35em
}

.desktop-button a {
  padding: 0.5em;
  color: #163f6e;
  text-decoration: none;
  display: block;
}

.desktop-button .icon {
  vertical-align: baseline;
}

.mobile-button {
  color: #163f6e !important;
}
```

double check that you have stylings for both desktop and mobile since the functionality will look & feel different.